### PR TITLE
Added new filters, pagination and stylized Admin Dashboard

### PR DESCRIPTION
**Pull request by varshasivakumar**
1. Add filters for Admin staff - GPA range, application number, student first and last name, application status, program
2. Removed the clickable tab on admin dashboard
3. Reverse mapping for gender on admin detail application page
4. Stylized "no applications exist"
5. Admin Staff can see the school he is associated with on the dashboard
6. Added pagination (for applications > 10) for admin staff dashboard view
7. Stylized the admin dashboard application cards

### CODE REVIEW

File: admissions/advfilters.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import django_filters
+
+from application.models import HighSchoolApplication
+
+FILTER_CHOICES = (("0", "Rejected"), ("1", "Accepted"), ("2", "Pending"))
```
Comment (snehalkenjale) [inline admissions/advfilters.py]: can we also show WithDrawn status on the admin side, and also hide Accept/reject buttons if the student withdraws his application

Comment (varshasivakumar) [reply admissions/advfilters.py]: Done

**Comment by varshasivakumar** [comment]
> Everything looks good and the filtering is great! however, there is no validation being performed on the GPA filter. You can enter values that are not between 0 and 4.

Included validations for min - max GPA, invalid GPA etc.

File: admissions/templates/admissions/index.html [hunk 2]
```diff
@@ -1,37 +1,149 @@
 {% extends "dashboard/index.html" %}
 {% load static %}
+{% load widget_tweaks %}
+{% load mytags %}
+
 {% block content %}
+    <link rel="stylesheet" type="text/css"
+          href="{% static 'admissions/admissions.css' %}">
+    <script type="application/javascript">
+        function validate() {
```
Comment (snehalkenjale) [inline admissions/templates/admissions/index.html]: can you move this code to .js file?

Comment (varshasivakumar) [reply admissions/templates/admissions/index.html]: Done

**Pull request merged**